dependencies.py

```diff
@@ -124,6 +126,26 @@ def _install_dependencies(dependencies, target_dir):
         print("Installed %s into %s" % (dependency.name, target_dir))
 
 
+def _get_python_interpreter():
+    """Return the path to the python3 interpreter.
+
+    Under linux sys.executable is set to the python3 interpreter used by Qgis.
+    However, under Windows/Mac this is not the case and sys.executable refers to the
+    Qgis start-up script.
+    """
+    interpreter = None
+    executable = sys.executable
+    head, tail = os.path.split(executable)
+    if tail == "python3":
```

Comment:
python3.7 ofzo zou het ook kunnen zijn:
```suggestion
    if "python3" in filename:
```